recipes/uwsgi/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "2.0.12" %}
+
+package:
+  name: uwsgi
+  version: '{{ version }}'
+
+source:
+  fn: "uwsgi-{{ version }}.tar.gz"
+  url: "https://pypi.io/packages/source/u/uWSGI/uwsgi-{{ version }}.tar.gz"
+  md5: 1451cab954bad0d7d7429e4d2c84b5df
+  patches:
+    - darwin.patch  # [osx]
+    # ^^ darwin.patch from https://github.com/unbit/uwsgi/commit/cf62d9f8ca6c9d0d64eaddb45534c8db27978af3
+    #    remove for version >2.0.12
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+    skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    # - gcc      # [unix]
```

Comment:
It would be nice if we could have a standard way for the conda-forge recipes to denote when a compiler is required.  I'll update the PR with a proposal.